packages/plugin-ext/src/plugin/plugin-context.ts

```diff
@@ -916,6 +917,10 @@ export function createAPIFactory(
             registerDocumentHighlightProvider(selector: theia.DocumentSelector, provider: theia.DocumentHighlightProvider): theia.Disposable {
                 return languagesExt.registerDocumentHighlightProvider(selector, provider, pluginToPluginInfo(plugin));
             },
+            /** @stubbed proposed API */
+            registerMultiDocumentHighlightProvider(selector: theia.DocumentSelector, provider: theia.MultiDocumentHighlightProvider): theia.Disposable {
```

Comment:
If we want to make this work in the future, we should mark it with `@monaco-uplift`.

Reply:
I added this annotation here: https://github.com/eclipse-theia/theia/blob/d692875e197897d08e32c5a6d3844b931179ef41/packages/plugin/src/theia.proposed.multiDocumentHighlightProvider.ts#L77
Should I also add it to this doc?

Reply:
I try to do as few changes to the api.d.ts files we take from VS Code because it makes it easier to maintain them, IMO (diff, etc.). Unless this is a general convention, I would prefer the `@monaco-uplift` annotatio where we put the stub implementation.

Reply:
Thanks for explanation, @tsmaeder. 
I have rebased and updated my changes, as this is a trivial fix and there was the usual conflict with the changelog. The monaco uplift annotation has been moved to plugin-context file.